src/mapandenum/task1/service/StudentService.java

```diff
@@ -0,0 +1,99 @@
+package mapandenum.task1.service;
+
+import mapandenum.task1.enumerations.Faculty;
+import mapandenum.task1.model.Student;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class StudentService {
+
+    //initialize a list of students
+    public static List<Student> initStudentList() {
+
+        List<Student> students = new ArrayList<>();
+
+        //create some students and add them to the list if all arguments are legal
+        try {
+            Student student1 = new Student("Diana", "Asatryan", 22,
+                    "098811111", Faculty.MATH);
+            students.add(student1);
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+        }
+
+        try {
+            Student student2 = new Student("Diana", "Asatryan", 32,
+                    "0922111111", Faculty.PHYSICS);
+            students.add(student2);
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+        }
+
+        try {
+            Student student3 = new Student("Diana", "Asatryan", 21,
```

Comment:
in cases when you have try/catch similar block for each added student, you can create addStudentInList method and have try/catch block only there